_data/communication.yml

```diff
@@ -13,6 +13,13 @@ websites:
       sms: Yes
       doc: https://help.37signals.com/sign-in/questions/460-how-do-i-set-up-and-use-phone-verification-for-logging-in
 
+    - name: Carbon Black Community Exchange
+      url: https://community.carbonblack.com/
+      img: cb.png
```

Comment:
Also, would it be possible to rename the image to "carbonblack.png", so that it is clear which site the image is used for?